04. Working with study/Curation script/README.md

```diff
@@ -23,14 +23,15 @@ The script takes as input the accessions of Genestack study or studies which sho
 You must also supply a rules file, which specifies the rules for mapping.
 
 ```bash
-odm-curate-study --rules <rules.json> <study accession(s)>
+odm-curate-study --rules <rules.json> <study accession(s)> -H GENESTACK_ENDPOINT_ADDR
 ```
+Where `GENESTACK_ENDPOINT_ADDR` is the URL of the Genestack platform.
 
 You can specify a template for the curation with the --template parameter. If not specified the default template for the
 organisation is used.
 
 ```bash
-odm-curate-study --template <template accession>
+odm-curate-study --template <template accession> -H GENESTACK_ENDPOINT_ADDR
```

Comment:
let's maybe add here --rules <rules.json> <study accession(s)>? cos anyway without mentioning rules file and study accession this example won't work